### ISSUE TITLE
Refine domain and range of "has driver" and "has disease driver"

### ISSUE THREAD

**Issue by matentzn** [oborel/obo-relations#541]
See https://github.com/oborel/obo-relations/pull/531

**Comment by nlharris** [comment]
Since #531 was merged, I think this can be closed?

**Comment by matentzn** [comment]
No, I added this ticket after the PR was merged to indicate that we omitted this. This ticket will need to be taken care of in a wider effort of consolidating disease relationships, but I don't see that for 2022. Maybe I am wrong.
